<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedConfigurationOptions</class>
 <widget class="QDialog" name="AdvancedConfigurationOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced options</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="setupTab">
        <attribute name="title">
         <string>Setup</string>
        </attribute>
        <layout class="QVBoxLayout" name="setupLayout"/>
       </widget>
       <widget class="QWidget" name="viewsTab">
        <attribute name="title">
         <string>Views</string>
        </attribute>
        <layout class="QVBoxLayout" name="viewsLayout"/>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
